keras_nlp/utils/text_generation.py

```diff
@@ -26,15 +26,23 @@ def _validate_prompt(prompt):
     return prompt
 
 
-def _validate_token_probability_fn(token_probability_fn, prompt):
-    """Helper function to validate token probability fn output."""
+def _validate_token_probability_fn(
```

Comment:
probably better to just have a consistent return type, and ignore it in functions that don't need it